pkgs/desktops/gnome-2/desktop/gopenvpn/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, fetchFromGitHub, pkgconfig, glib, gtk2, gnome2, dbus_glib, gmime, libnotify, libgnome_keyring,
+  openssl, cyrus_sasl, gnonlin, sylpheed, gob2, gettext, intltool, libglade, polkit, autoconf, automake, expect,
+  openvpn, makeWrapper, pkexecPath ? "/var/setuid-wrappers/pkexec" }:
+
+let
+  redirects = [ "/usr/bin/pkexec=${pkexecPath}" ];
+in stdenv.mkDerivation rec {
+  rev = "2b9285f1ea2d11434b8ee8e10d937b2ea5285b63";
+  version = "0.8";
+  name = "gopenvpn-${version}";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "proteansec";
+    repo = "gopenvpn";
+    sha256 = "0pkdravr5wm7phdr2rd1rx1ry4dpwkrf0hv7wi99n4ywsl5jqwq8";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ glib gtk2 dbus_glib gmime libnotify libgnome_keyring openssl cyrus_sasl gnonlin sylpheed gob2
+                  gettext intltool libglade polkit pkgconfig autoconf automake expect openvpn makeWrapper ];
```

Comment:
You can use `autoreconfHook` here. It probably also make the `expect` hack  and the rest of `preConfigure` unnecessary